spec/rubocop/cop/lint/useless_access_modifier_spec.rb

```diff
@@ -237,6 +238,82 @@
     end
   end
 
```

Comment:
Can you add a simple test for `it 'should not report rails-related offenses if Rails is not enabled'`?